docs/layout.md

```diff
@@ -1,38 +0,0 @@
-## Using layouts
-
-### TL;DR;
-
-Use
-
-```
-// default flex-direction: row
-// Put items next to each other
-<Flex />
-```
-
-### Flex
-
-```
-import { Flex } from 'app/components/Layout';
-```
-
-If you need wrapping, use
-
-```
-<Flex wrap />
-// or
-<Flex wrapReverse />
-
-// flex-direction: column
-// Stack children vertically
-<Flex column />
-```
-
-and put stuff inside them.
-
-They support the props `alignItems`, `justifyContent`, `padding`, `flex` and
-`width`, which should be sufficient for most purposes (the default values are
-usually fine).
-
-These components are only responsible for how items are laid out, and have no
-opinions on colors etc. They are just `div` elements with flex styling.
```

Comment:
Some of there are quite nice (and still relevant) actually

Reply:
I agree that they are nice to have, but if they are just laying around as an md file, how often would they be used? We could of course leave them around, but imo they have little use until we get a system and habits for frontend docs up and running.

Reply:
@LudvigHz Do you really think so? I feel like everyone looks at the current code, and not the docs. Have you ever looked at these before today? I say we just delete them, and can `git-revert` them back if you ever need them